dissect/target/plugins/apps/editor/windowsnotepad.py

```diff
@@ -94,15 +91,27 @@
 };
 """
 
-WINDOWS_SAVED_TABS_EXTRA_FIELDS = [("datetime", "modification_time"), ("digest", "hashes"), ("path", "saved_path")]
+GENERIC_TAB_CONTENTS_RECORD_FIELDS = [
+    ("string", "editor"),
+    ("string", "content"),
+    ("path", "path"),
+    ("string", "deleted_content"),
+    ("path", "source"),
```

Comment:
```suggestion
    ("string", "editor"),
    ("string", "content"),
    ("path", "path"),
    ("string", "deleted_content"),
    ("path", "source"),
```
Some of these seem to be duplicate from `COMMON_EDITOR_FIELDS`. Could you perhaps that record descriptor here?